README.md

```diff
@@ -927,6 +927,7 @@ API | Description | Auth | HTTPS | CORS |
 | [OpenUV](https://www.openuv.io) | Real-time UV Index Forecast | `apiKey` | Yes | Unknown |
 | [OpenWeatherMap](https://openweathermap.org/api) | Weather | `apiKey` | Yes | Unknown |
 | [Storm Glass](https://stormglass.io/) | Global marine weather from multiple sources | `apiKey` | Yes | Yes |
+| [US Weather ](https://api.weather.gov/points/47.7601,-122.2054) | Weather for US by Latitude and Longitude | `apiKey` | Yes | Unknown |
```

Comment:
Is `apiKey` required to use this API? It doesn't look like, I can call the API with no API key.